src/js/components/tasks/cards/card.tsx

```diff
@@ -203,6 +245,14 @@ const OrgCard = ({
         onRequestClose={closeAssignUserModal}
         setUser={doAssignUser}
       />
+      {reviewOrgOutOfDate && (
+        <RefreshOrgModal
+          orgUrl={window.api_urls.scratch_org_redirect(org?.id)}
+          missingCommits={orgCommitIdx}
+          isOpen={refreshOrgModalOpen}
```

Comment:
I think we should probably hide the modal while refreshing, so maybe this could change to:

```suggestion
          isOpen={refreshOrgModalOpen && !isRefreshingOrg}
```